keps/sig-network/4559-kubelet-probes/README.md

```diff
@@ -132,35 +124,13 @@ and startup probes (listed based on severity):
   This led to the creation of [readiness gates] to provide better pod
   readiness information.
 
-- **Exec Probe Performance (?)**: [Exec probes are much, much slower than
-  network probes]. Apparently this slowness is a somewhat inevitable
-  side effect of how exec probes work in an OCI runtime context?
-  Possibly because it requires making multiple synchronous calls? No known
-  issues were opened outside of the blogpost but anyway, it's possible
-  this could be improved.
-
-- **Defining localhost probes doesn't work as expected**: When defining
+6. **Defining localhost probes doesn't work as expected**: When defining
   `.Host` to be `127.0.0.1` the user might be expecting this to be confined
   to the pod's own localhost but it is not. The entire node can be exposed
   by doing so which can be surprising to the users.
 
 See further discussion in [kubernetes #102613].
 
-#### RESOLVED Summary discussions (preserving for history)
-- Figure out if there are any concerns with non-namespace-based Linux
-  runtimes (kata, kubevirt, etc): There are no concerns here because both kata containers and [kubevirt] use network namespaces. They expect the CNI to set up the pod networking. Then their agent creates the required networking infrastructure inside this pod network namespace to connect to the VM interface
```

Comment:
Both of our remaining approaches delegate to CRI, so it wouldn't matter even if kata and kubevirt did do weird things, since their CRI implementations would know how to deal with that.